src/tools/gui/stlink-gui.c

```diff
@@ -518,7 +516,7 @@ stlink_gui_set_connected (STlinkGUI *gui)
 }
 
 static void
-connect_button_cb (GtkWidget *widget, gpointer data)
+connect_button_cb (GtkWidget * UNUSED(widget), gpointer data)
```

Comment:
This is way ugly. You can just do it in a more portable way (pseudo-C):

<function> (param1, param2) {
     (void)param1;
     param2++;
}